build/lib/extensions.js

```diff
@@ -414,19 +414,14 @@ async function webpackExtensions(taskName, isWatch, webpackConfigLocations) {
     const webpackConfigs = [];
     for (const { configPath, outputRoot } of webpackConfigLocations) {
         const configOrFnOrArray = require(configPath);
-        function addConfig(configOrFn) {
-            let config;
-            if (typeof configOrFn === 'function') {
-                config = configOrFn({}, {});
+        function addConfig(configOrFnOrArray) {
+            for (const configOrFn of Array.isArray(configOrFnOrArray) ? configOrFnOrArray : [configOrFnOrArray]) {
+                const config = typeof configOrFn === 'function' ? configOrFn({}, {}) : configOrFn;
+                if (outputRoot) {
+                    config.output.path = path.join(outputRoot, path.relative(path.dirname(configPath), config.output.path));
+                }
                 webpackConfigs.push(config);
```

Comment:
I checked the webpack configs. None of them pass functions, which is good, because this code double-pushed function configs.
Nothing sets `outputRoot` that I could see, so the `if (outputRoot)` code is just-in-case. I left it because I don't know what it's supposed to do.